11-change-ppp-mtu-debug.sh

```diff
@@ -30,17 +30,18 @@ while true; do
           echo $pinterface is one we should be checking
           # Check to see if we need to update the config file
           echo Checking MTU for $pinterface
-          pmtu=$(grep $(($PTARGET-8)) /etc/ppp/peers/$pinterface)
+          pmtu=$(grep 'mtu $(($PTARGET))' /etc/ppp/peers/$pinterface)
           if [[ $pmtu ]]; then
+            echo MTU already correct in /etc/ppp/peers/$pinterface
+            pmtucorrect=1
+          else
             echo Current config file MTU for $pinterface is $pmtu
             echo Updating config file for $pinterface
             echo Making changes to /etc/ppp/peers/$pinterface
             # Update MTU in ppp interface config file
-            sed -i 's/ '$(($PTARGET-8))'/ '$PTARGET'/g' /etc/ppp/peers/$pinterface
-            killall -SIGHUP pppd
-          else
-            echo MTU already correct in /etc/ppp/peers/$pinterface
-            pmtucorrect=1
+            sed -i 's/mtu\s[0-9]*/mtu '$PTARGET'/g' /etc/ppp/peers/$pinterface
+            sed -i 's/mru\s[0-9]*/mru '$PTARGET'/g' /etc/ppp/peers/$pinterface
```

Comment:
instead of doing a ptarget-8 replacement i changed this to replace the number value of mtu and mru lines individually no matter what value they have. for aforementioned ptarget-8 problems